crowdsourcing/viewsets/project.py

```diff
@@ -78,12 +78,17 @@ def destroy(self, request, *args, **kwargs):
 
 class ModuleViewSet(viewsets.ModelViewSet):
 
+    def list(self, request, *args, **kwargs):
+        return Response(None, status=status.HTTP_403_FORBIDDEN)
+
     def get_queryset(self):
         queryset = Module.objects.all()
         requesterid=self.request.query_params.get('requesterid',None)
         if requesterid is not None:
             queryset = Module.objects.all().filter(owner__id=requesterid)
-        return queryset
+        # return queryset
+        return Response(None, status=status.HTTP_403_FORBIDDEN)
```

Comment:
no need for this, DRF handles it, we just set the permission_classes=[IsAuthenticated]